repository.rst

```diff
@@ -213,18 +213,19 @@ returned by ``go env GOPATH``.
 | Whether Gazelle should generate build files in the repository. In ``"auto"``                            |
 | mode, Gazelle will run if there is no build file in the repository root                                 |
 | directory.                                                                                              |
-+--------------------------------+----------------------+-------------------------------------------------+
-| :param:`build_config`          | :type:`label`        | :value:`@//:WORKSPACE`                          |
-+--------------------------------+----------------------+-------------------------------------------------+
++------------------------+----------------+---------------------------------------------------------------+
```

Comment:
Moving these characters causes the table cells to be misaligned. You can preview this by going to "..." > View File here in code review.

Since you need extra space here, you'll need to push out the right side of the table to keep everything aligned. Sorry this is an annoying, manual process. ReStructuredText doesn't handle tables well. We use it because it does a better job than markdown, which doesn't allow paragraphs within cells.